src/ui/FormWidgets/TextInput.ts

```diff
@@ -64,6 +65,7 @@ export class TextInput implements IFormWidget, IFormWidgetSettable {
     if (currentValue != '') {
       this.onChange(this);
     }
+    this.lastQueryText = this.getInput().value;
```

Comment:
Why is this line for? Should it be unit tested?

Reply:
Yes, it can be tested.